documentation/assemblies/tracing/assembly-distributed-tracing.adoc

```diff
@@ -9,65 +9,71 @@ Distributed tracing allows you to track the progress of transactions between app
 
 In Strimzi, tracing facilitates the end-to-end tracking of messages: from source systems to Kafka, and then from Kafka to target systems and applications. It complements the metrics that are available to view in link:{BookURLDeploying}#assembly-metrics-setup-{context}[Grafana dashboards], as well as the component loggers.
 
-[discrete]
-== How Strimzi supports tracing
-
-Support for tracing is built in to the following components:
+Support for tracing is built in to the following Kafka components:
 
 * Kafka Connect
 * MirrorMaker
 * MirrorMaker 2.0
 * Strimzi Kafka Bridge
 
-Strimzi uses the OpenTracing and Jaeger Tracing projects to implement tracing.
-Tracing is enabled by setting the `.spec.tracing.type` property to `jaeger` in the component configuration.
+Tracing is not supported for Kafka brokers.
+You enable and configure tracing for these components using template configuration properties in their custom resources. 
+
+You specify a tracing type using the `tracing.type` property:
+
+* `opentelemetry` for OpenTelemetry with the Jaeger Exporter
+* `jaeger` for OpenTracing with Jaeger
```

Comment:
The tracing is still enabled in `.spec.tracing.type`. Only the additional configuration options for the tracer are in the `.spec.template` section.

Reply:
Changed as follows:

> You enable and configure tracing for these components through their custom resources. 
> You enable tracing by specifying a tracing type using the `spec.tracing.type` property:
> 
> * `opentelemetry` for OpenTelemetry with the Jaeger Exporter
> * `jaeger` for OpenTracing with Jaeger
> 
> You add tracing configuration using `spec.template` properties.